### ISSUE TITLE
IP150+ module do support SWAN disable - only local LAN network

### ISSUE THREAD

**Issue by wili98w** [ParadoxAlarmInterface/pai#356]
Hi just want to let you know, that there are special code that you can enter durring the IP 150+ module configuraiton, which allows you to use the module in LAN mode, without forcing or using SWAN servers. 
In that mode, you are loosing ability to control the panel trough a WEB interface of the IP150+ module itself, but still you can manage the panel trough Paradox Gold app (and also through babyware and reprogramming)
It is working perfectly fine with HAAS as well. (Currently running on EVO192 with 6.5 - not encrypted FW)
Now upgrade request, no additional configuration of 150+ needed

Copying the screenshot of it
![image](https://user-images.githubusercontent.com/106034385/228909855-4b88928e-ae2a-49fe-9557-d3ce687094db.png)

So this is a dedicated firmware v5.06.016 for ip150+ with support with all the new panels and firmware.

The only thing you should do is to enter "special firmware access code "IP150PCN" durring the upgrade process of IP150+.
It is possible from inField and Paradox Gold App.
![image](https://user-images.githubusercontent.com/106034385/228911586-9e00432f-bc9f-4516-9c43-352ce4164777.png)
![image](https://user-images.githubusercontent.com/106034385/228911614-f29d0b41-9496-4bb0-aeb4-e5c5f15ee27a.png)

in the Paradox Gold App is available in installer section
![image](https://user-images.githubusercontent.com/106034385/228911768-7d9caee5-71bf-4e05-be51-4bc39157112e.png)

Statement from paradox!
Use the "IP150PCN" beta key to downgrade the module to v.5.06.016 for closed network operation.
v5.06.016 has a web interface for network setup. When SWAN polling is disabled, there is no trouble for network connection in closed networks.

**Comment by wili98w** [comment]
Some screenshots from the ip150+ gui
![image](https://user-images.githubusercontent.com/106034385/228912513-292156ab-31ed-4b37-bd5c-54b8cd8b0dea.png)
![image](https://user-images.githubusercontent.com/106034385/228912753-9c5783ac-cde4-4351-9fe2-ddb0cddbf202.png)

**Comment by yozik04** [comment]
Thank you @wili98w

**Comment by mharizanov** [comment]
Thanks @wili98w

**Comment by robertoamd90** [comment]
On my module the update remain to 0%, 
any idea?
thanks

**Comment by robertoamd90** [comment]
> On my module the update remain to 0%, i'm on App any idea? thanks

i have resolved using BabyWare + In-Field.

Thanks!

**Comment by budix1** [comment]
Hi, i am waiting for instalation Paradox alarm in my house. I have EVOHD+ and IP150+ . It is possible change to local without supplier software or what should I ask them to make it work for me? ? I'm already confused by this, I was expecting a simple integration.... thanks

**Comment by thibault-78** [comment]
Hello,
@wili98w many thanks. It perfectly worked for me (I was able to disable SWAN-> closed network)
I did not success to connect thought your method, I had to use: Static IP

**Comment by gfplab-scaffardi** [comment]
Hi! Thanx for the info but... i have an IP150 with bootloader 2.14.000 that requires a specific old version of infield. So i cannot access that firmware. Is there any way to have the same feature with my version of IP150? 

If not, do you confirm that buying and updating in this way an IP150+ makes it useable in the same time with PAI and Insite Gold app?

Thank you!

**Comment by yozik04** [comment]
Not possible to use PAI and Insite Gold at the same time.

**Comment by iliantihov** [comment]
I can't access the IP150+ web interface anymore. Does anyone know how I can set up email notifications via BabyWare or some other way?

**Comment by yozik04** [comment]
If PAI is stopped then nothing can interfere. It is IP150+ bug then.

**Comment by J-Prince** [comment]
> If PAI is stopped then nothing can interfere. It is IP150+ bug then.

Yes, definitely not PAI's problem. I was just hoping to hear from others using that particular firmware version if they had the same issue with the web interface.

**Comment by Zemenv** [comment]
This post fixed my system, many thanks for it. I downgraded the firmware and knock on wood it is working. Good job.

**Comment by padiyoyo** [comment]
Hi, I'm having the same problem for months. The PAI add-ons kept disconnecting by itself, with multiple restarts. I want to try downgrading the IP150 firmware. How do I get babyware software? The official paradox website requires registration but we have to be an installer, not customer. Any advice?

**Comment by Benoit3** [comment]
Tested successfully with IP150+MQ

**Comment by yozik04** [comment]
@Benoit3, can you please elaborate more? What you have tested?

**Comment by Benoit3** [comment]
For several years (4?) , I was using pai with a now fifteen year old IP100 and an EVO192 V6.86. The email feature of the IP100 was not anymore usable with today’s email smtp server, which are now requesting more robust security protocols : I ordered an IP150+ but received an IP150+MQ :-(. But the IP150+MQ software can be downgraded to V5.06.016 with the procedure described above, so it works perfectly .

Thank you for the pai development

**Comment by Benoit3** [comment]
I wished to add a column in the [compatibility table ](https://github.com/ParadoxAlarmInterface/pai/wiki/Compatibility) to add a column IP150+MQ, but I'm not enough Markdown skilled to simply add a column...

**Comment by yozik04** [comment]
There is a note on the compatibility page how to edit. But new column might not fit in the limited width of the table.

**Comment by Benoit3** [comment]
I saw it but I don't managed to copy paste existing markdown table without corrupting it

**Comment by mogim0gi** [comment]
Is this still working ?

**Comment by DenadaPatron** [comment]
Can confirm this still works and I bought the IP150+MQ with the newest version on it (V6.15.008) 

However, it wasn't an easy process as I needed to first receive an approval to log-in to paradox.com by distributor (just used conventional sign-up process) after a few days I managed to add my SP7000+ as a site and receive SWAN e-mail. With the account I was able to download most recent BabyWare 5.6.68 and etsablish a connection via Site Token or Site ID (can't remember now). BabyWare now has a built-in InField application which I have reached my IP150+MQ from and managed to downgrade with the provided code. Afterwards connection via StaticIP worked via BabyWare but there was some back and forth in the process (had to disable SP7000+ from AC and Battery to reset memory, arm and disarm from the control panel, reset IP150+MQ via reset button). Also, blocked WAN traffic on the IP150+MQ.

However, my webui stopped working alltogether but BabyWare, PAI and physical controll panel is more than enough for me.


Thank you wili98w for providing this solution and Benoit3 for giving me faith that I can get this working on IP150+MQ.

2026-07-11 21:18:07,077 - INFO     - PAI.paradox.paradox - Initiating panel connection
2026-07-11 21:18:07,102 - INFO     - PAI.paradox.hardware.spectra_magellan.panel - Installer login
2026-07-11 21:18:07,131 - INFO     - PAI.paradox.hardware.spectra_magellan.panel - Authentication Success
2026-07-11 21:18:07,132 - INFO     - PAI.paradox.paradox - Connection OK
2026-07-11 21:18:07,133 - INFO     - PAI.paradox.paradox - Loading data from panel memory
2026-07-11 21:18:07,133 - INFO     - PAI.paradox.hardware.panel - Loading definitions
2026-07-11 21:18:07,134 - INFO     - PAI.paradox.hardware.panel - Updating Definitions from Panel
2026-07-11 21:18:07,322 - INFO     - PAI.paradox.hardware.panel - Zone definitions loaded (0.19s)
2026-07-11 21:18:07,420 - INFO     - PAI.paradox.hardware.panel - Pgm definitions loaded (0.1s)
2026-07-11 21:18:07,422 - INFO     - PAI.paradox.hardware.panel - Loading labels
2026-07-11 21:18:07,422 - INFO     - PAI.paradox.hardware.panel - Updating Labels from Panel

**Comment by DenadaPatron** [comment]
> > If PAI is stopped then nothing can interfere. It is IP150+ bug then.
> 
> Yes, definitely not PAI's problem. I was just hoping to hear from others using that particular firmware version if they had the same issue with the web interface.

I have this exact issue but regardless managed to connect to the IP150+MQ via BabyWare and PAI

**Comment by yozik04** [comment]
Maybe Web stops working when PAI is connected? Did you try to stop it?
